packages/react-provider/src/components/FluentProvider/renderFluentProvider.tsx

```diff
@@ -14,7 +15,9 @@ export const renderFluentProvider = (state: FluentProviderState, contextValues:
       <ThemeContext.Provider value={contextValues.theme}>
         <ThemeClassNameContext.Provider value={contextValues.themeClassName}>
           <TooltipContext.Provider value={contextValues.tooltip}>
-            <slots.root {...slotProps.root}>{state.root.children}</slots.root>
+            <TextDirectionProvider dir={contextValues.textDirection}>
```

Comment:
As Griffel can't have dependencies on Fluent contexts - it has it's own provider to get text direction.